features/bounding-box-zoeken.feature

```diff
@@ -0,0 +1,131 @@
+# language: nl
+
+Functionaliteit: Als gebruiker wil ik een pand of adresseerbaar object kunnen zoeken binnen een gebied
```

Comment:
Ik zou voor de titel van de functionaliteit geen user story gebruiken. De user story beschrijft volgens mij in het kort waarom een stakeholder de functionaliteit wil

Reply:
ik pas het aan